circle/functions

```diff
@@ -674,7 +674,8 @@ update_minideb_derived() {
         exit 1
     fi
 
-    REPO_TO_UPDATE=$1
+    # Append '.git' so TravisCI git version can understand the URL
+    REPO_TO_UPDATE="${1}.git"
```

Comment:
this should be done only if the URL does not already have it or we could end up with clone urls such as `https://path/to/repo.git.git`.

perhaps you could use

```
REPO_TO_UPDATE="${1%.git}.git"
```

Reply:
I was looking exactly for that! We decided it was not critical enough but with that simple expression I think it is worth it